cspc-operator/examples/cspc/cspi.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: cstor.openebs.io/v1
+kind: CStorPoolInstance
+metadata:
+  name: cspi-stripe
+  namespace: openebs
+spec:
+  pools:
```

Comment:
`CStorPoolInstance` spec does not have `pools` field and each CSPI will have only one pool spec. Please update the yaml with correct fields.